Assets/Scripts/PlayerManager.cs

```diff
@@ -1,17 +1,4 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-
-public class PlayerManager : MonoBehaviour {
-    public GameObject HumanPrefab;
-    public GameObject BotPrefab;
-    public GameObject OffScreenUIPrefab;
-    // Controls whether we need to spawn player 1
-    public bool isPlayerInScene = true;
-    public Transform player1SpawnPoint;
-    public Transform player2SpawnPoint;
-    public ScoreBoard[] scoreBoards;
-    private GameObject player1, player2;
+﻿public static class Util {
```

Comment:
Might as well either rename this script to utils or something. To be addressed next time.

Reply:
o i forgot to rename the file oops